README.rst

```diff
@@ -28,29 +28,46 @@ Usage
 
     from aiocassandra import aiosession
     from cassandra.cluster import Cluster
+    from cassandra.query import SimpleStatement
 
+    # connection is blocking call
     cluster = Cluster()
+    # aiocassandra uses executor_threads to talk to cassndra driver
+    # https://datastax.github.io/python-driver/api/cassandra/cluster.html?highlight=executor_threads
     session = cluster.connect()
 
-    # best way is to use cassandra prepared statements
-    cql = session.prepare('SELECT now() FROM system.local;')
 
-    @asyncio.coroutine
-    def main(*, loop):
-        # patches and adds `execute_future` to `cassandra.cluster.Session`
-        aiosession(session, loop=loop)
-        return (yield from session.execute_future(cql))
+    async def main(*, loop):
+        # patches and adds `execute_future`, `execute_future` and `prepare_future`
+        # to `cassandra.cluster.Session`
+        aiosession(session)
```

Comment:
loop=loop

Reply:
Is it mandatory? It looks like an optional param. And it's only recommended to bypass loop explicitly in tests, not normal code. Still, in such example `main(*, loop)` also needs changes b/c you get an unused here.
/cc: @hellysmile